src/validator.js

```diff
@@ -7,14 +7,16 @@ import promisify from 'es6-promisify';
 
 import { terminalWidth } from 'cli';
 import * as constants from 'const';
+import { CHROME_MANIFEST, INSTALL_RDF, MANIFEST_JSON } from 'const';
 import * as exceptions from 'exceptions';
 import * as messages from 'messages';
-import { getPackageTypeAsString,
-         checkMinNodeVersion,
+import { checkMinNodeVersion,
          gettext as _, singleLineString } from 'utils';
 
 import log from 'logger';
 import Collector from 'collector';
+import InstallRdfParser from 'parsers/installrdf';
```

Comment:
Again, as above, I'd think the `Validator` would be dealing only with `Scanner` classes, while the `Scanner` classes would use a `Parser` class internally to process files.

Maybe it's just a semantic thing and these should be called `Scanner`s and not `Parser`s.